frontend/www/js/omegaup/components/arena/EphemeralGrader.vue

```diff
@@ -5,7 +5,11 @@
     :problem="problem"
     :can-submit="canSubmit"
     :can-run="canRun"
-  />
+  >
+    <template #zip-buttons>
+      <slot name="zip-buttons"></slot>
+    </template>
```

Comment:
Almost done, just a minor change.

Both components, `Details.vue` and `Detailsv2.vue` are the only ones that call this component. And, in both components the buttons should be hidden. So, you can do:
```suggestion
    <template #zip-buttons><div></div></template>
```
here, and the changes in the other two components (`Details.vue` and `Detailsv2.vue`) can be reverted

Reply:
I have committed the changes. My original idea was to use this element as a wrapper around actual IDE, this would let us do all kinds of initializations on the store or props, like we with `initialLanguage`. So the same element was going to be used when constructing full IDE, but we can change that a bit.